server/src/main/java/org/opensearch/search/aggregations/bucket/terms/GlobalOrdinalsStringTermsAggregator.java

```diff
@@ -99,8 +99,8 @@ public class GlobalOrdinalsStringTermsAggregator extends AbstractStringTermsAggr
     protected final ResultStrategy<?, ?, ?> resultStrategy;
     protected final ValuesSource.Bytes.WithOrdinals valuesSource;
 
-    private final LongPredicate acceptedGlobalOrdinals;
```

Comment:
do we need to change anything in this class since we aren't extending it anymore?

Reply:
Reverted, all the changes left now are just rename and comment.